go/task.go

```diff
@@ -80,9 +80,9 @@ func (t *Task) MemProt(addr, size uint64, prot int) error {
 func (t *Task) MemUnmap(addr, size uint64) error {
 	addr, size = align(addr, size, true)
 	for _, mm := range t.memsim.Mem.FindRange(addr, size) {
```

Comment:
Why run this in a loop if we're doing the same thing (unchanged `addr` and `size`) every time?

Reply:
we were talking in slack, I landed a different patch instead